src/ui/server/helpers/map-eligibility-answers/index.ts

```diff
@@ -1,15 +1,28 @@
-import FIELD_IDS from '../../constants/field-ids/insurance';
+import { FIELD_IDS, TOTAL_CONTRACT_VALUE } from '../../constants';
 import { SubmittedDataInsuranceEligibility } from '../../../types';
 
-const { BUYER_COUNTRY_ISO_CODE } = FIELD_IDS.ELIGIBILITY;
+const {
+  INSURANCE: {
+    ELIGIBILITY: { BUYER_COUNTRY_ISO_CODE, TOTAL_CONTRACT_VALUE_ID },
+  },
+} = FIELD_IDS;
+
+export const mapTotalContractValue = (answer: boolean) => {
+  if (answer) {
+    return TOTAL_CONTRACT_VALUE.LESS_THAN_500K.DB_ID;
+  }
+
+  return TOTAL_CONTRACT_VALUE.MORE_THAN_500K.DB_ID;
+};
 
 const mapEligibilityAnswers = (answers: SubmittedDataInsuranceEligibility) => {
   if (answers.buyerCountry) {
-    const { buyerCountry, ...otherAnswers } = answers;
+    const { buyerCountry, wantCoverOverMaxAmount, ...otherAnswers } = answers;
 
     const mapped = {
       ...otherAnswers,
       [BUYER_COUNTRY_ISO_CODE]: buyerCountry?.isoCode,
+      [TOTAL_CONTRACT_VALUE_ID]: mapTotalContractValue(Boolean(wantCoverOverMaxAmount)),
```

Comment:
this would be better in a const and then passed to `mapTotalContractValue`

Reply:
Updated